x-pack/solutions/security/plugins/security_solution/public/common/components/security_route_page_wrapper/index.tsx

```diff
@@ -102,11 +102,18 @@ export const SecurityRoutePageWrapper: FC<PropsWithChildren<SecurityRoutePageWra
 export const withSecurityRoutePageWrapper = <T extends {}>(
   Component: React.ComponentType<T>,
   pageName: SecurityPageName,
-  redirectOnMissing?: boolean
+  {
+    redirectOnMissing,
+    redirectIfUnauthorized,
+  }: { redirectOnMissing?: boolean; redirectIfUnauthorized?: boolean } = {}
 ) => {
   return function WithSecurityRoutePageWrapper(props: T) {
     return (
-      <SecurityRoutePageWrapper pageName={pageName} redirectOnMissing={redirectOnMissing}>
+      <SecurityRoutePageWrapper
+        pageName={pageName}
+        redirectOnMissing={redirectOnMissing}
+        redirectIfUnauthorized={redirectIfUnauthorized}
```

Comment:
Do we need both?

Reply:
Well, I understand that there is a separate logic for each of these, just one was not being passed down. 
What do you think?